src/lib/checkbox/_checkbox-theme.scss

```diff
@@ -63,7 +80,23 @@
   }
 
   // TODO(jelbourn): remove style for temporary ripple once the real ripple is applied.
-  .md-checkbox-focused .md-ink-ripple {
-    background-color: md-color($accent, 0.26);
+  .md-checkbox-focused {
+    &.md-primary {
+      .md-ink-ripple {
+        background-color: md-color($primary, 0.26);
+      }
+    }
```

Comment:
Done if you meant `&.md-primary` instead of `&.primary`.
`MdButton` also uses the CSS classes `md-primary`, `md-accent` and `md-warn`.